src/webgpu/api/validation/createRenderPipeline.spec.ts

```diff
@@ -422,24 +422,27 @@ g.test('pipeline_output_targets')
   - The componentCount of the fragment output (e.g. f32, vec2, vec3, vec4) must not have fewer
     channels than that of the color attachment texture formats. Extra components are allowed and are discarded.
 
+  Otherwise, color state write mask must be 0.
+
   MAINTENANCE_TODO: update this test after the WebGPU SPEC ISSUE 50 "define what 'compatible' means
   for render target formats" is resolved.`
   )
   .params(u =>
     u
       .combine('isAsync', [false, true])
+      .combine('writeMask', [0, 0xf])
       .combine('format', kRenderableColorTextureFormats)
```

Comment:
see https://github.com/gpuweb/gpuweb/issues/2770. I think we should also test writeMasks that are not "all"